addons/mail/models/mail_channel.py

```diff
@@ -477,8 +477,8 @@ def is_current_user(r):
                     },
                 },
             ])
-        notification = _('%s started a live conference', partner.name)
-        self.message_post(body=notification, message_type="notification")
+        notification = _("%s started a live conference", partner.name if partner else guest.name)
```

Comment:
`partner.name or guest.name` should do the job.

Note that just this change would be more easily merged as a separate PR.